istio-telemetry/kiali/templates/configmap.yaml

```diff
@@ -14,6 +14,14 @@ data:
       pilot: {{ .Values.global.configNamespace }}
       prometheus: {{ .Values.global.prometheusNamespace }}
     istio_namespace: {{ .Values.global.istioNamespace }}
+    auth:
+      strategy: {{ .Values.dashboard.auth.strategy }}
```

Comment:
should be `.Values.kiali...`

Reply:
Thanks @howardjohn, completely missed that!